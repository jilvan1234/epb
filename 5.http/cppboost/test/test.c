//=========
// main.c
//=========

void main()
{
	FILE *infile;
	infile = fopen("callbacks-list.c", "r");
	if (infile == NULL) 
	{
		printf("file open error!\n");
		return;
	}

	const char delim[] = " ,;(){}\t\n\r'";
	char line[LenLong];
	char str[LenLong];
	while (!feof(infile))
	{
		if (fgets(line, LenLong, infile)!= NULL)
		{
			printf("%s", line);
			printf("--------------------------------------------------\n");
			strcpy(str, line);
			StrSplit_Gen(str);
			printf("==================================================\n");
		}
	}
	fclose(infile);

	char teststr[LenShort];
	sprintf(teststr,"%%s=\"%%s\"\n");
	printf(teststr, "xyz", "123");
	
	char *ppp=(char *)zz00000000();
	printf(ppp, "xxxxxxxxxxx"); printf("\n");
	printf((char *)zz00000000(), "yyyyyyyyyyy"); printf("\n");
	printf((char *)zz00000000(), "123456"); printf("\n");
	printf("##################################################\n");
}