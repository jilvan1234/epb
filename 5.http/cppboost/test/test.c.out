#include "AddShell.h"
//=========
// main.c
//=========

void main()
{
	FILE *infile;
	infile = fopen(z00000000(), z00000001());
	if (infile == NULL) 
	{
		printf(z00000002());
		return;
	}

	const char delim[] = z00000003();
	char line[LenLong];
	char str[LenLong];
	while (!feof(infile))
	{
		if (fgets(line, LenLong, infile)!= NULL)
		{
			printf(z00000004(), line);
			printf(z00000005());
			strcpy(str, line);
			StrSplit_Gen(str);
			printf(z00000006());
		}
	}
	fclose(infile);

	char teststr[LenShort];
	sprintf(teststr,z00000007());
	printf(teststr, z00000008(), z00000009());
	
	char *ppp=(char *)zz00000000();
	printf(ppp, z00000010());
	printf((char *)zz00000000(), z00000011());
	printf((char *)zz00000000(), z00000012());
	printf(z00000013());
}